agent/acs/handler/attach_eni_handler_test.go

```diff
@@ -28,9 +28,9 @@ import (
 )
 
 const (
-	eniMessageId = "123"
-	randomMAC    = "00:0a:95:9d:68:16"
-	waitTimeout  = 10
+	eniMessageId      = "123"
+	randomMAC         = "00:0a:95:9d:68:16"
+	waitTimeoutMillis = 10
```

Comment:
It was bugging me that `waitTimeout` was set to `10` without any unit information. `GoRename` seemed simpler than changing all lines where this was referred to. Honestly, I don't think the name of this variable is that important in the larger context of this PR. I'm sticking with what I have now unless some one feels way too strongly about it.